src/utils/isString.ts

```diff
@@ -0,0 +1,3 @@
+export function isString(value: any) {
```

Comment:
Will `typeof value === 'string' not work?

Reply:
Actually I just found out lodash has an `isString` method so let's use that lol